js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let navAnchors = document.querySelectorAll('a');
+let mainHeader = document.querySelector('h1');
+let headerButton = document.querySelector('button');
+let headerImg = document.getElementById('cta-img');
+let H4 = document.querySelectorAll("h4");
+let paragraphs = document.querySelectorAll("p")
+let middleImg = document.getElementById("middle-img");
```

Comment:
Yus! I like how you're keeping your selectors at the top of the file here - nice and organised.